{'description': {'en': 'collection of ebuilds by ycarus'},
 'homepage': 'http://gpo.zugaina.org/Overlays/zugaina/',
 'name': 'zugaina',
 'owner': [{'email': 'ycarus@zugaina.org'}],
 'quality': 'experimental',
 'source': [{'type': 'rsync',
             'uri': 'rsync://gentoo.zugaina.org/zugaina-portage'}],
 'status': 'unofficial'}
$ pmaint sync zugaina
WARNING:pkgcore:repository at '/home/mgorny/sync/steveeJ' has unsupported profile format(s): 'profile-set'

receiving incremental file list

Number of files: 3,386 (reg: 2,792, dir: 594)
Number of created files: 0
Number of regular files transferred: 0
Total file size: 8.71M bytes
Total transferred file size: 0 bytes
Literal data: 0 bytes
Matched data: 0 bytes
File list size: 68.50K
File list generation time: 0.001 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 673
Total bytes received: 76.97K

sent 673 bytes  received 76.97K bytes  17.25K bytes/sec
total size is 8.71M  speedup is 112.20
*** syncing zugaina
*** synced zugaina
 * Sync succeeded
$ pmaint regen --use-local-desc --pkg-desc-index -t 1 zugaina
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'

!!! ERROR: net-news/feed2imap-0.7 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '177' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.7.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.7.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.7.ebuild'
!!!           feed2imap-0.7.ebuild, line  10:  Called inherit 'ruby'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'ruby'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for ruby failed


!!! ERROR: net-news/feed2imap-0.9.3 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '187' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.9.3.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.9.3.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.9.3.ebuild'
!!!         feed2imap-0.9.3.ebuild, line  11:  Called inherit 'ruby'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'ruby'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 187' for eclass ruby

received unknown com: export CATEGORY='net-news' PR=r0 PVR='0.9.4' PV='0.9.4' EBUILD='/home/mgorny/repos/zugaina/net-news/feed2imap/feed2imap-0.9.4.ebuild' P='feed2imap-0.9.4' PF='feed2imap-0.9.4' PN=feed2imapfailed
!!! caught exception inherit requires an unknown eclass, ruby cannot be found while processing ebuild src: net-news/feed2imap-0.7
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-news/feed2imap-0.9.3
!!! caught exception inherit requires an unknown eclass, ruby cannot be found while processing ebuild src: net-news/feed2imap-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-news/feed2imap-9999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: games-strategy/bos-2.0
Traceback (most recent call last):
  File "/home/mgorny/pkgcore-venv/bin/pmaint", line 4, in <module>
    scripts.main(basename(__file__))
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/scripts/__init__.py", line 43, in main
    commandline.main(subcommands)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/util/commandline.py", line 915, in main
    exitstatus = main_func(options, out, err)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/scripts/pmaint.py", line 215, in regen_main
    eclass_caching=(not options.disable_eclass_caching))
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/__init__.py", line 68, in _recast_exception_decorator
    return functor(*args, **kwds)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/repo.py", line 236, in _cmd_api_regen_cache
    self._get_observer(observer), threads=threads, **options)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 41, in regen_repository
    regen_iter(passthru(repo), _get_repo_helper(), observer)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 529, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe
 * Cache regen failed with 1
